includes/wikia/nirvana/WikiaView.class.php

```diff
@@ -204,7 +206,20 @@ private function getTemplateOptions( $controllerClass, $methodName, $controllerB
 		return $templates;
 	}
 
-	protected function getTemplateAnnotation( $controllerClass, $methodName ) {
+	private function getClassName( string $controllerClassWithNamespace ): string {
```

Comment:
then it could not run the class, I need to remove namespace only to build template path